### PR TITLE
[SD-3211],[SD-3210] Tests and UI change to allow ingest of email from google forms

### PR DESCRIPTION
**Pull request by marwoodandrew**
This PR is dependent on the pr superdesk/superdesk-core#15

### CODE REVIEW

**Comment by sivakuna-aap** [comment]
:+1:

**Comment by akintolga** [comment]
:+1:

**Comment by mdhaman** [comment]
:+1:

**Comment by amagdas** [comment]
Reference to core needs to be updated so the tests will pass.

**Comment by marwoodandrew** [comment]
Can I do that before the core PR is merged?

File: client/app/scripts/superdesk-ingest/views/settings/emailConfig.html [hunk 1]
```diff
@@ -29,6 +29,10 @@
       ng-change="$parent.setConfig(provider)">
 </div>
 <div class="field">
+      <label for="email-formatted">Formatted Email Parser</label>
```
Comment (plamut) [inline client/app/scripts/superdesk-ingest/views/settings/emailConfig.html]: Translate?

**Comment by amagdas** [comment]
Core PR is merged.

**Pull request merged**